### PR TITLE
#116 Switching to new ruby hash syntax and other minor syntax changes

### PR DESCRIPTION
**Pull request by rennanoliveira**

### CODE REVIEW

**Comment by tauil** [comment]
:+1:

**Comment by tauil** [comment]
@lucianosousa , @aoqfonseca and @viniciussbs review this, please.

File: app/models/event.rb [hunk 1]
```diff
@@ -37,30 +39,30 @@ def opened
...
     end
 
     def occurs_first
-      order("occurs_at ASC")
+      order('occurs_at ASC')
     end
 
     def active
-      where('occurs_at >= ?', Date.today)
+      where('occurs_at >= ?', Time.zone.today)
```
Comment (lucianosousa) [inline app/models/event.rb]: Why this change?

Comment (tauil) [inline app/models/event.rb]: As long as I know, this is needed for a proper work with time zone.

Comment (rennanoliveira) [inline app/models/event.rb]: Using Date.today will bring you the machine's datetime, which might differ from your actual datetime (with time zones)

Usually I switch from Date.today to Time.zone.today or Date.current.
source: https://robots.thoughtbot.com/its-about-time-zones

Though in this case Date.current might be better, due to Time's [limitations](http://stevenyue.com/2013/03/23/date-time-datetime-in-ruby-and-rails/):

Comment (lucianosousa) [inline app/models/event.rb]: Nice.
We will merge this soon as possible. Thanks for the contribution

**Comment by lucianosousa** [comment]
Thanks!
